src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -288,6 +288,20 @@ namespace winrt::TerminalApp::implementation
         {
             auto newTabImpl = winrt::make_self<TerminalTab>(pane);
             _InitializeTab(newTabImpl);
+
+            if (_autoPeer)
+            {
+                // we can't check if this is a leaf pane,
+                // but getting the profile returns null if we aren't, so that works!
+                if (const auto profile{ pane->GetProfile() })
+                {
+                    _autoPeer.RaiseNotificationEvent(
+                        Automation::Peers::AutomationNotificationKind::ActionCompleted,
+                        Automation::Peers::AutomationNotificationProcessing::ImportantMostRecent,
+                        fmt::format(std::wstring_view{ RS_(L"NewTabAnnouncement") }, profile.Name()),
+                        L"NewTab" /* unique name for this notification category */);

```

Comment:
does this fire a hundred times if you do "nt;nt;nt;nt;nt" from the command palette?

Reply:
Actually, it tries but it keeps interrupting itself. Let me update the `AutomationNotificationProcessing` enum here

This also exposed another problem. I didn't realize that `commandline mode` has a history too! I'll go ahead and add that in.

Reply:
oh wait... that second point is for a different a11y PR. My bad! haha (I should still check that though)